docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,371 +9,131 @@ to save power and free CPU resources.
-   +----------------------------------+-------------------------+---------------+-----------------------------------------------------------------+
-   | ``GNA_FIRMWARE_MODEL_IMAGE``     | ``string``              | ``""``        | Sets the name for the embedded model binary dump file.          |
-   +----------------------------------+-------------------------+---------------+-----------------------------------------------------------------+
-   | ``GNA_PRECISION``                | ``I16``, ``I8``         | ``I16``       | Sets the preferred integer weight resolution for quantization   |
-   |                                  |                         |               | (ignored for models produced using POT).                        |
-   +----------------------------------+-------------------------+---------------+-----------------------------------------------------------------+
-   | ``PERF_COUNT``                   | ``YES``, ``NO``         | ``NO``        | Turns on performance counters reporting.                        |
-   +----------------------------------+-------------------------+---------------+-----------------------------------------------------------------+
-
-   The parameters are passed as strings to `IECore.load_network <api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.load_network>`_.
-
-   Normally, you do not need to select the execution target (``GNA_EXEC_TARGET``) and compilation target (``GNA_COMPILE_TARGET``). The default value for the execution target corresponds to available hardware, or latest hardware version supported by the plugin (i.e., GNA 3.0) if there is no GNA HW in the system. The compilation target is the same as the execution target by default. However, you may want to change the targets, for example, if you want to check how a model compiled for one generation would behave on the other generation (using the SW emulation mode), or if you are willing to export a model for a specific version of GNA HW.
-
-   You can change the ``GNA_DEVICE_MODE`` parameter at run time by sending a configuration dict to the `IECore.load_network <api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.load_network>`_ call, which works for any value excluding ``GNA_SW_FP32``. This enables you to switch the execution between software emulation mode and hardware execution mode after the model is loaded.
+Please refer to [Stateful models] (@ref openvino_docs_IE_DG_network_state_intro) for more details about such models.
 
-@endsphinxdirective
-## How to Interpret Performance Counters
+> **NOTE**: Typically, GNA is used in streaming scenarios, when minimizing the latency is important. Taking into account that POT does not support the `TensorIterator` operation, the recommendation is to use the `--transform` option of the Model Optimizer to apply `LowLatency2` transformation when converting an original model.
```

Comment:
could you please provide MO cmd with `LowLatency2 ` usage? or maybe via C++ / Python code

Reply:
@ilya-lavrenov There is a line above to page with statefull models. This page contains information about LowLatency2 as well.